src/IECoreArnold/Renderer.cpp

```diff
@@ -1031,6 +1032,7 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 			m_lightShader = attribute<IECoreScene::ShaderNetwork>( g_arnoldLightShaderAttributeName, attributes );
 			m_lightShader = m_lightShader ? m_lightShader : attribute<IECoreScene::ShaderNetwork>( g_lightShaderAttributeName, attributes );
 			substituteShaderIfNecessary( m_lightShader, attributes );
+			m_muteLight = optionalAttribute<bool>( g_muteLightAttributeName, attributes );
```

Comment:
Same as for Cycles, I don't think `optional` is buying us anything here.

Reply:
Fixed in 61a0719ffbda738276335ec4bfb8bfeb9476ab95